### PR TITLE
Reuse LocalAlluxioCluster to speedup FileSystemMaster integration test

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
Jenkins, test this please

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java [hunk 1]
```diff
@@ -4646,4 +4646,12 @@ public AlluxioURI reverseResolve(AlluxioURI ufsUri) throws InvalidPathException
     }
     return resolution.getUri();
   }
+
+  /**
+   * @return username of root of inode tree, null if the inode tree is not initialized
```
Comment (ZacBlanco) [inline core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java]: username of the owner? I don't entirely understand

File: minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java [hunk 2]
```diff
@@ -107,7 +107,7 @@ public void restartMasters() throws Exception {
   /**
    * Configures and starts the proxy.
    */
-  private void startProxy() throws Exception {
+  public void startProxy() throws Exception {
```
Comment (ZacBlanco) [inline minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java]: what's the reason to need to change this?

**Comment by ZacBlanco** [comment]
jenkins, test this please

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**